MIGRATING.md

```diff
@@ -8,7 +8,8 @@
     - `createCardToken()` now requires `CardParams` instead of `Card`
 - Changes to `ConfirmPaymentIntentParams`
     - `returnUrl` has been removed as a parameter on `ConfirmPaymentIntentParams` static creation
-       methods. The SDK manages setting this value internally.
+       methods. The SDK manages setting this value internally.  It will use an installed browser
+       with Custom Chrome Tab support, or, if not available, a WebView.
```

Comment:
I think the sentence you added is not directly to `returnUrl`. Maybe it should be a separate bullet. Also I think there is more nuance here and we could include the comments / table in the docstring of `Stripe#confirmPayment`.

Reply:
See if these updates are better.